docbook/pom.xml

```diff
@@ -106,10 +111,55 @@
                     </options>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>process-spring-version</id>
+                        <goals>
+                            <goal>exec</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <executable>${project.basedir}/scripts/set_spring_dep_ver.sh</executable>
+                    <environmentVariables>
+                        <BASEDIR>${project.basedir}</BASEDIR>
+                    </environmentVariables>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>dev.jbang</groupId>
+                <artifactId>jbang-maven-plugin</artifactId>
+                <version>0.0.4</version>
+                <executions>
+                    <execution>
+                        <id>run</id>
+                        <phase>process-resources</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <jbangVersion>${jbang.ver}</jbangVersion>

```

Comment:
If I remove the version, it will start to download the latest ver(which is 0.47.1 at this time), but the download will fail and gives network error:

![image](https://user-images.githubusercontent.com/201907/111728176-514f4e80-88a7-11eb-8324-1e118d44c1d7.png)

Reply:
Interesting. I guess we should change it the version to `0.67.3` as it the latest then. The plugin was probably picking up the version I had installed already.

Reply:
okay!